### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by ritikbanger**
Enhanced the Doc by removing grammatical errors.

### CODE REVIEW

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ It's the most universal and [the most popular](https://www.npmtrends.com/core-js
 
 `core-js` is my own hobby project and it does not bring me any profit. It takes too much time and it is really costly: to finish work on `core-js@3`, I had left my job some months ago. This project facilitates the life of many people and companies. For these reasons, it makes sense to start raising funds to support the maintenance of `core-js`.
 
-If you interested in the `core-js` project or use it in your day-to-day work, you can become a sponsor on **[Open Collective](https://opencollective.com/core-js#sponsor)** or **[Patreon](https://www.patreon.com/zloirock)**.
+If you are interested in the `core-js` project or use it in your day-to-day work, you can become a sponsor on **[Open Collective](https://opencollective.com/core-js#sponsor)** or **[Patreon](https://www.patreon.com/zloirock)**.
 
 You can propose [me](http://zloirock.ru/) a good job where I will be able to work on something related.
 
-Or you can contribute in another way: you can help improving code, tests or documentation (currently, `core-js` documentation is terrible!).
+Or you can contribute in another way: you can help improve code, tests or, documentation (currently, `core-js` documentation is terrible!).
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:24]: This comma is not required.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 2]
```diff
@@ -128,16 +128,16 @@ Another popular feature request was support for the [`.forEach` method on DOM co
 
 #### No more non-standard non-proposed features
 
-Many years ago, I started writing a library which I needed as the core of my JavaScript applications: this library contained polyfills and some utilities for common needs. After some time, it was published as `core-js`. I think that at this moment most `core-js` users do not use non-standard `core-js` features. Almost all of them were removed in previous releases, and it's time to remove all the remaining ones from `core-js`. Starting from this release, `core-js` can be finally called a polyfill.
+Many years ago, I started writing a library that I needed as the core of my JavaScript applications: this library contained polyfills and some utilities for common needs. After some time, it was published as `core-js`. I think that at this moment most `core-js` users do not use non-standard `core-js` features. Almost all of them were removed in previous releases, and it's time to remove all the remaining ones from `core-js`. Starting from this release, `core-js` can be finally called a polyfill.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:131]: It makes no sense.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 3]
```diff
@@ -128,16 +128,16 @@ Another popular feature request was support for the [`.forEach` method on DOM co
...
 
-### Packages, entry points and modules names
+### Packages, entry points, and modules names
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:133]: This comma is not required.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 4]
```diff
@@ -128,16 +128,16 @@ Another popular feature request was support for the [`.forEach` method on DOM co
...
+- [`core-js-pure`](https://www.npmjs.com/package/core-js-pure), which provides polyfills without polluting the global environment. It's the equivalent of `core-js/library` from `core-js@2`. (~440KB)
 - [`core-js-bundle`](https://www.npmjs.com/package/core-js-bundle): a bundled version of `core-js` which defines global polyfills.
 
-In previous versions of `core-js`, modules with polyfills for stable ECMAScript features and ECMAScript proposals were prefixed with `es6.` and `es7.` respectively. It was a decision taken in 2014 when all the features which could be after ES6 were considered as ES7. In `core-js@3` all stable ECMAScript features are prefixed with `es.`, while ECMAScript proposals with `esnext.`.
+In previous versions of `core-js`, modules with polyfills for stable ECMAScript features and ECMAScript proposals were prefixed with `es6.` and `es7.` respectively. It was a decision taken in 2014 when all the features that could be after ES6 were considered as ES7. In `core-js@3` all stable ECMAScript features are prefixed with `es.`, while ECMAScript proposals with `esnext.`.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:140]: The same, makes no sense.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 5]
```diff
@@ -128,16 +128,16 @@ Another popular feature request was support for the [`.forEach` method on DOM co
...
 
-### Packages, entry points and modules names
+### Packages, entry points, and modules names
 
-A popular issue was the big size (~2MB) of the `core-js` package and duplication of many of its files. For this reason, `core-js` was split into three packages:
+A popular issue was a big size (~2MB) of the `core-js` package and duplication of many of its files. For this reason, `core-js` was split into three packages:
 - [`core-js`](https://www.npmjs.com/package/core-js), which defines global polyfills. (~500KB, [40KB minified and gzipped](https://bundlephobia.com/result?p=core-js@3.0.0-beta.20))
-- [`core-js-pure`](https://www.npmjs.com/package/core-js-pure), which provides polyfills without pollution the global environment. It's the equivalent of `core-js/library` from `core-js@2`. (~440KB)
+- [`core-js-pure`](https://www.npmjs.com/package/core-js-pure), which provides polyfills without polluting the global environment. It's the equivalent of `core-js/library` from `core-js@2`. (~440KB)
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:137]: I would prefer "pollution" more.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 6]
```diff
@@ -176,13 +176,13 @@ import "core-js/stage/2";
...
+For almost all users, for optimization of `core-js` import, I recommend using [`babel`](#Babel). However, for some cases still a useful [`core-js-builder`](http://npmjs.com/package/core-js-builder). Now it supports the `targets` argument which takes a [`browserslist`](https://github.com/browserslist/browserslist) query with target engines - you can create a bundle that contains only required for target engines polyfills. For cases like this, I made the [`core-js-compat`](http://npmjs.com/package/core-js-compat) package, more info about it you could find in [`@babel/preset-env` part of this article](#babelpreset-env).
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:185]: `a` makes no sense, it's better to completely rewrite this sentence. I don't see a sense in changing `which` -> `that` in this case too.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 7]
```diff
@@ -194,7 +194,7 @@ As mentioned above, `babel` and `core-js` are tightly integrated: `babel` gives
 
 ### `@babel/polyfill`
 
-[`@babel/polyfill`](https://babeljs.io/docs/en/next/babel-polyfill.html) is a wrapper package which only includes imports of stable `core-js` features (in Babel 6 it also included proposals) and `regenerator-runtime/runtime`, needed by transpiled generators and async functions. This package doesn't make it possible to provide a smooth migration path from `core-js@2` to `core-js@3`: for this reason, it was decided to deprecate `@babel/polyfill` in favor of separate inclusion of required parts of `core-js` and `regenerator-runtime`.
+[`@babel/polyfill`](https://babeljs.io/docs/en/next/babel-polyfill.html) is a wrapper package that only includes imports of stable `core-js` features (in Babel 6 it also included proposals) and `regenerator-runtime/runtime`, needed by transpiled generators and async functions. This package doesn't make it possible to provide a smooth migration path from `core-js@2` to `core-js@3`: for this reason, it was decided to deprecate `@babel/polyfill` in favor of separate inclusion of required parts of `core-js` and `regenerator-runtime`.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:197]: The same.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 8]
```diff
@@ -315,13 +315,13 @@ Until Babel 7.3, `useBuiltIns: usage` was unstable and not fully reliable: many
 
 I improved the techniques used to determine which polyfills should be added on property accesses, object destructuring, `in` operator, global object property accesses.
 
-`@babel/preset-env` now injections polyfills required for syntax features: iterators when using `for-of`, destructuring, spread and `yield` delegation; promises when using dynamic `import`, async functions and generators, etc.
+`@babel/preset-env` now injections polyfills required for syntax features: iterators when using `for-of`, destructuring, spread and `yield` delegation; promises when using dynamic `import`, async functions, and generators, etc.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:318]: This comma is also not required.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 9]
```diff
@@ -336,7 +336,7 @@ import _includesInstanceProperty from "@babel/runtime-corejs3/core-js-stable/ins
 _includesInstanceProperty(array).call(array, something);
 ```
 
-Another notable change is the support of ECMAScript proposals. By default, `@babel/plugin-transform-runtime` does not inject polyfills for proposals and use entry points which do not include them but, exactly as you can do in `@babel/preset-env`, you can set the `proposals` flag to enable them: `corejs: { version: 3, proposals: true }`.
+Another notable change is the support of ECMAScript proposals. By default, `@babel/plugin-transform-runtime` does not inject polyfills for proposals and use entry points that do not include them but, exactly as you can do in `@babel/preset-env`, you can set the `proposals` flag to enable them: `corejs: { version: 3, proposals: true }`.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:339]: The same.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 10]
```diff
@@ -366,35 +366,35 @@ myArrayLikeObject[Symbol.iterator] = Array.prototype[Symbol.iterator];
...
 
-The biggest missing important feature are property descriptors: when they aren't available, some features can't be polyfilled because they either are accessors (like `RegExp.prototype.flags` or `URL` properties setters) or are accessors-based (like typed arrays polyfill). In order to workaround this lack, we need to use different workarounds (for example, to keep `Set.prototype.size` updated). Maintenance of those workarounds sometimes is too painful, and removing them would highly simplify many polyfills.
+The biggest missing important feature is property descriptors: when they aren't available, some features can't be polyfilled because they either are accessors (like `RegExp.prototype.flags` or `URL` properties setters) or are accessors-based (like typed arrays polyfill). In order to work around this lack, we need to use different workarounds (for example, to keep `Set.prototype.size` updated). Maintenance of those workarounds sometimes is too painful, and removing them would highly simplify many polyfills.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:381]: `workaround` should not be split.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 11]
```diff
@@ -366,35 +366,35 @@ myArrayLikeObject[Symbol.iterator] = Array.prototype[Symbol.iterator];
...
 
 ### Extended web standards support?
 
 `core-js` is currently focused on ECMAScript support, but it also supports a few web standards features which are available cross-platform and closely related to ECMAScript. Adding polyfills for web standards like `fetch` is a very popular feature request.
 
-The main reason why `core-js` doesn’t include them was that it would have seriously increased bundles size and it would have forced `core-js` users to load features which might not have been needed. Now `core-js` is maximally modular, user can include only some chosen features, there are tools like `@babel/preset-env` and `@babel/runtime` which helps to get rid of unused or unnecessary polyfills.
+The main reason why `core-js` doesn’t include them was that it would have seriously increased bundles size and it would have forced `core-js` users to load features that might not have been needed. Now `core-js` is maximally modular, user can include only some chosen features, there are tools like `@babel/preset-env` and `@babel/runtime` which helps to get rid of unused or unnecessary polyfills.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:397]: The same.

File: docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md [hunk 12]
```diff
@@ -424,9 +424,9 @@ TC39 is working really hard to improve ECMAScript: you can see the progress by l
...
+Polyfilling of built-in modules, [as stated by the authors of the proposal](https://github.com/tc39/proposal-javascript-standard-library/issues/2), only means falling back to layered APIs or import maps. This means that if a native module will be missing, it will be possible to load a polyfill from a provided URL. That's absolutely not what polyfills need, and it is incompatible with the architecture of `core-js` and every other popular polyfill project. Import maps shouldn't be the only way to polyfill built-in modules.
 
-We will be able to get a built-in module just by usage ES modules syntax with a special prefix. This syntax haven't any equal based on the previous version of the language - transpiled modules will not be able to interact with not transpiled in modern engines - it will cause problems for package distribution.
+We will be able to get a built-in module just by using ES modules syntax with a special prefix. This syntax haven't equal based on the previous version of the language - transpiled modules will not be able to interact with not transpiled in modern engines - it will cause problems for package distribution.
```
Comment (zloirock) [inline docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md:429]: `any` shouldn't be removed.

**Review by zloirock — CHANGES_REQUESTED** [review]
Thanks, but some fixes are required.

**Review by zloirock — APPROVED** [review]

**Comment by ritikbanger** [comment]
Thanks for merging!

**Pull request merged**